test/system/practice/not_logged_in_test.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+require 'application_system_test_case'
+
+class Practice::NotLoggedInTest < ApplicationSystemTestCase
+  test 'set ogp image to meta-tag when exists ogp image' do
+    practice = practices(:practice1)
+    visit_with_auth edit_practice_path(practice), 'komagata'
+    within 'form[name=practice]' do
+      attach_file 'practice[ogp_image]', 'test/fixtures/files/practices/ogp_images/1.jpg', make_visible: true
+    end
+    click_button '更新する'
+    logout
+
+    visit practice_path(practice)
+    ogp_image = File.basename find('meta[property="og:image"]', visible: false)['content']
+    twitter_card_image = File.basename find('meta[name="twitter:image"]', visible: false)['content']
+    assert_equal '1.jpg', ogp_image
+    assert_equal '1.jpg', twitter_card_image
+  end
+
+  test 'set default ogp image to meta-tag when do not exists ogp image' do
+    practice = practices(:practice1)
+
+    visit practice_path(practice)
+    ogp_image = File.basename find('meta[property="og:image"]', visible: false)['content']
```

Comment:
```suggestion
    ogp_image = File.basename(find('meta[property="og:image"]', visible: false)['content'])
```

どのメソッドの引数が何なのかわかりづらいのでこういう場合はカッコをつけたほうがわかりやすいかもです。

Reply:
該当部分にカッコを追加しました🙋‍♂️ 確認をお願いします🙇‍♂️

https://github.com/fjordllc/bootcamp/blob/caafe35c9f19980229e149193a12ec07248b4fdf/test/system/practice/not_logged_in_test.rb#L16-L17

https://github.com/fjordllc/bootcamp/blob/caafe35c9f19980229e149193a12ec07248b4fdf/test/system/practice/not_logged_in_test.rb#L26-L27